reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md

```diff
@@ -49,67 +49,17 @@ various package managers:
 | CentOS, RedHat |`sudo yum install -y powershell` | `sudo yum install -y powershell-preview`|
 | Fedora   |`sudo dnf install -y powershell` | `sudo dnf install -y powershell-preview`|
 
-## Ubuntu 16.04
+## Ubuntu 16.04 LTS (Xenial Xerus) and 18.04 LTS (Bionic Beaver)
 
-### Installation via Package Repository - Ubuntu 16.04
+### Installation via Package Repository - Ubuntu 16.04 and 18.04
 
 PowerShell Core for Linux is published to package repositories for easy installation and updates.
 
 The preferred method is as follows:
 
 ```sh
 # Download the Microsoft repository GPG keys
-wget -q https://packages.microsoft.com/config/ubuntu/16.04/packages-microsoft-prod.deb
-
-# Register the Microsoft repository GPG keys
-sudo dpkg -i packages-microsoft-prod.deb
-
-# Update the list of products
-sudo apt-get update
-
-# Install PowerShell
-sudo apt-get install -y powershell
-
-# Start PowerShell
-pwsh
-```
-
-As superuser, register the Microsoft repository once. After registration, you can update
-PowerShell with `sudo apt-get upgrade powershell`.
-
-### Installation via Direct Download - Ubuntu 16.04
-
-Download the Debian package `powershell_6.2.0-1.ubuntu.16.04_amd64.deb` from the [releases][] page
-onto the Ubuntu machine.
-
-Then, in the terminal, execute the following commands:
-
-```sh
-sudo dpkg -i powershell_6.2.0-1.ubuntu.16.04_amd64.deb
-sudo apt-get install -f
-```
-
-> [!NOTE]
-> The `dpkg -i` command fails with unmet dependencies. The next command, `apt-get install -f`
-> resolves these issues then finishes configuring the PowerShell package.
-
-### Uninstallation - Ubuntu 16.04
-
-```sh
-sudo apt-get remove powershell
-```
-
-## Ubuntu 18.04
-
-### Installation via Package Repository - Ubuntu 18.04
-
-PowerShell Core for Linux is published to package repositories for easy installation and updates.
-
-The preferred method is as follows:
-
-```sh
-# Download the Microsoft repository GPG keys
-wget -q https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb
+wget -q https://packages.microsoft.com/config/ubuntu/$(. /etc/os-release && echo $VERSION_ID)/packages-microsoft-prod.deb
```

Comment:
Please revert this type of change.  Unfortunately these need to stay hard coded for synchronization with other teams.

Reply:
Basically don't try to guess which one is correct.